pip/tests-integration/test_base_qir.py

```diff
@@ -109,18 +110,19 @@ def check_call(i: int, name: str, num_args: int) -> None:
     check_call(10, "__quantum__qis__h__body", 1)
     check_call(11, "__quantum__qis__s__body", 1)
     check_call(12, "__quantum__qis__s__adj", 1)
-    check_call(13, "__quantum__qis__t__body", 1)
-    check_call(14, "__quantum__qis__t__adj", 1)
-    check_call(15, "__quantum__qis__x__body", 1)
-    check_call(16, "__quantum__qis__y__body", 1)
-    check_call(17, "__quantum__qis__z__body", 1)
-    check_call(18, "__quantum__qis__swap__body", 2)
-    check_call(19, "__quantum__qis__cx__body", 2)
-    check_call(20, "__quantum__qis__m__body", 2)
+    check_call(13, "__quantum__qis__sx__body", 1)
```

Comment:
Does this mean that whatever executes this QIR should now handle SX?

Reply:
Technically, QIR does not proscribe a particular gate list, but this does add it to the "canonical" set of gates used by the QDK as mentioned in the PR description. This works with the service and is handled by other tools like qir-runner: https://github.com/qir-alliance/qir-runner/pull/221.